src/app/features/group-list/group-item/group-item.component.ts

```diff
@@ -0,0 +1,21 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+
+export interface GroupItem {
+  id: number;
+  name: string;
+  description: string;
+}
```

Comment:
to można wyrzucić do oddzielnego pliku i przy okazji dodać plik `index.ts` i dodać tam wszystkie exportu z danego folderu

Reply:
Wyciągnięty plik z interfejsem, dodany plik index.ts